### DISCUSSION TITLE
Using for over the air update in pico sdk

### DISCUSSION THREAD

**Discussion by isarwar93** [rhulme/pico-flashloader#16] [Ideas]
Hey,

I want to make a project, where I would like to update the pico over the air, may be using some http server or something similar. Am I allowed to use your code? (of course with credits and your repo link)

With Kind Regards,

Ismail

**Comment by rhulme** [comment]
Hi, yes, no problem.  The code is licensed under the 3-clause BSD license so there's no problem using it in your own project.
